smac/optimizer/objective.py

```diff
@@ -35,8 +35,8 @@ def _runtime(config: Configuration, run_history: RunHistory,
         instance_seed_pairs = run_history.get_runs_for_config(config)
 
     runtimes = []
-    for i, r in instance_seed_pairs:
-        k = RunKey(id_, i, r)
+    for i, r, b in instance_seed_pairs:
```

Comment:
These are definitively no instance-seed pairs any more, but rather instance-seed-budget pairs. I'm really not sure how to call these now, `instance_seed_budget_triplet`?

Reply:
This issue obviously also applies to the whole runhistory.